install.md

```diff
@@ -3,7 +3,7 @@
 ```sh
 oc new-app https://github.com/<project>/cluster-support-bot.git \
   -e APP_FILE=cluster-support-bot.py \
-  -e SLACK_SIGNING_SECRET=<credentials from https://api.slack.com/apps/XXXXX/general?> \
+  -e BOT_ID=<bot id -- hover over the bot name in slack to get this>
```

Comment:
nit: missing a trailing `\` to continue the line.

nittier nits: `id` -> `ID` and `slack` -> `Slack` ;)

Reply:
done